runtime/alloc.c

```diff
@@ -371,3 +371,13 @@ CAMLexport value caml_alloc_some(value v)
   Field(some, 0) = v;
   CAMLreturn(some);
 }
+
+CAMLprim value caml_atomic_make_aligned(value v)
+{
+  CAMLparam1(v);
+  const mlsize_t sz = Wosize_bhsize(Cache_line_bsize);
+  value res = caml_alloc_shr(sz, 0);
```

Comment:
Do you not want `sz - 1` here to account for the fact that we have a 1-word header? I remember that the earlier code did `caml_alloc_shr(7,0)`. What am I missing?

Reply:
Answering the question myself -- `Wosize_bhsize` and not `Wosize_bsize`! 👍